src/main/ruby/core/regexp.rb

```diff
@@ -192,7 +192,8 @@ def initialize(pattern, opts=nil, lang=nil)
   end
 
   def initialize_copy(other)
-    initialize other.source, other.options
+    Truffle.primitive :regexp_initialize_copy
+    raise PrimitiveFailure, "Regexp.clone(#{other.inspect}) primitive failed"
```

Comment:
When the method consists of just `Truffle.primitive` + `raise PrimitiveFailure`, I find it cleaner to just define a `@CoreMethod` instead.